tachyon/crypto/commitments/fri/two_adic_fri.h

```diff
@@ -345,10 +347,16 @@ class TwoAdicFRI {
          it != max_log_num_rows_for_point.end(); ++it) {
       const ExtF& point = it->first;
       uint32_t log_num_rows = it->second;
-      std::vector<ExtF> temp = base::Map(
-          absl::MakeSpan(subgroup.data(), (size_t{1} << log_num_rows)),
-          [&point](F x) { return ExtF(x) - point; });
-      CHECK(ExtF::BatchInverseInPlace(temp));
+      std::vector<ExtF> temp(size_t{1} << log_num_rows);
+      base::Parallelize(
+          temp, [&subgroup, &point](absl::Span<ExtF> chunk, size_t chunk_offset,
```

Comment:
Seems like `Base::Map()` is not used here. Please fix commit message.

Reply:
`temp` was constructed using `base::Map` before which is ran in serial.

```c++
      std::vector<ExtF> temp = base::Map(
          absl::MakeSpan(subgroup.data(), (size_t{1} << log_num_rows)),
          [&point](F x) { return ExtF(x) - point; });
```

Reply:
Oh I understood the commit message the wrong way lol.